2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 5 From line 1 to 25_
```python
import sklearn.metrics as metrics
import matplotlib.pyplot as plt
from matplotlib.style import use
import matplotlib.dates as mdates

# Evaluate the model
y_pred = model.predict(X_test)
mse = metrics.mean_squared_error(y_test, y_pred)
print(f'Mean Squared Error: {mse}')

prediction = {"date": y_test.index,
              "actual": y_test,
              "predicted": y_pred}
prediction = pd.DataFrame(prediction)

# Plot the prediction
use('ggplot')
fig, ax = plt.subplots()
prediction.plot.line(x = "date",
                     ax=ax)
ax.set_title("Actual vs Predicted")
ax.set_ylabel("Water level (m)")
ax.set_xlabel("Date")
ax.xaxis.set_major_formatter(mdates.DateFormatter('%Y-%m-%d'))
fig.tight_layout()
```
If you show the performance in the test data, I would at least add a few lines after saying something like "it seems the model has a decent performance, but how is it doing those predictions?" to smoothly transition into the three feature importance metrics. Showing the results without saying anything about them feels a bit weird to me.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
Done